Automattic/wp-calypso
# Remove redundant post normalizer rule: createBetterExcerptRefresh

During the Reader Refresh we experimented with creating post excerpts in a different way using the rule `createBetterExcerptRefresh`. We ultimately didn't use this new rule, so this PR removes it from the codebase.

### To test

Visit the Reader and ensure that excerpts are being created correctly in post cards:

<img width="828" alt="screen shot 2017-01-09 at 12 33 12" src="https://cloud.githubusercontent.com/assets/17325/21757329/d17ecffa-d667-11e6-8fa9-9dde5571803d.png">

Ensure that the post normalizer tests pass:

` npm run test-client client/lib/post-normalizer`
